libs/core/src/lib/util.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * @internal
+ */
+export function execRegex(regex: RegExp, val: string): string[] {
+  const arr = [];
+  let group: RegExpExecArray;
+
+  while ((group = regex.exec(val)) !== null) {
+    if (group.index === regex.lastIndex) {
+      regex.lastIndex++;
+    }
+
+    arr.push(...group);
+  }
+
+  return !arr.length ? null : arr;
```

Comment:
May be here return always arr, instead return null in case arr.length === 0

Reply:
This is done to copy same behavior of native `match/exec` methods, if nothing was found - `null` will be returned.